ps_facetedsearch.php

```diff
@@ -1506,4 +1578,52 @@ public function getWidgetVariables($hookName, array $configuration)
     {
         return [];
     }
+
+    /**
+     * Provides data about single template.
+     *
+     * @param int $idFilterTemplate ID of filter template
+     *
+     * @return array Filter data
+     */
+    public function getFilterTemplate($idFilterTemplate)
+    {
+        return $this->getDatabase()->getRow(
+            'SELECT *
+            FROM `' . _DB_PREFIX_ . 'layered_filter`
+            WHERE id_layered_filter = ' . $idFilterTemplate
```

Comment:
```suggestion
            WHERE id_layered_filter = ' . (int) $idFilterTemplate
```
Against SQL injection